components/AppContext.tsx

```diff
@@ -1,8 +1,10 @@
 import React from "react";
 
+type ColorScheme = 'light' | 'dark'
+
 const AppContext = React.createContext({
   token: "" as string | null | undefined,
-  scheme: "",
+  scheme: 'light' as ColorScheme,
```

Comment:
Should default to a valid colorsheme instead of `""`

Reply:
oh cool thx